aws/application/application.template

```diff
@@ -700,6 +704,8 @@ Resources:
           - !Ref CognitoUserPoolClaimForCrownCourtDefenceClient
           - !Ref CognitoUserPoolClientCrimeValidationService
           - !Ref CognitoUserPoolClientATSService
+          - !Ref CognitoUserPoolClientDcesReportService
+          - !Ref CognitoUserPoolClientDcesDrcReportService
```

Comment:
I think we have to recreate a new Cognito credential to pick up the new scope for DCES services. Please could you verify?

Reply:
I remember you saying about this before! As long as the resource server is created first then the client should know the scope exists. I will add a dependency to ensure the creation order.

Reply:
ah we already have the dependency - we should be good!